flink-table/flink-table-planner/src/main/scala/org/apache/flink/table/planner/codegen/calls/BuiltInMethods.scala

```diff
@@ -386,15 +386,29 @@ object BuiltInMethods {
     "timestampFloor",
     classOf[TimeUnitRange],
     classOf[Long],
+    classOf[Int],
     classOf[TimeZone])
 
+  val TIMESTAMP_FLOOR_TIME_ZONE_FOR_HIGH_PRECISiON = Types.lookupMethod(
```

Comment:
same comment about renaming.

Is there a reason to have a small `i` in the middle?